cmake/modules/hunter_copy_license_file.cmake

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2015, Aaditya Kalsi
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+function(hunter_copy_license_file srcdir dstfile)
+  # find the license file to copy
+  
+  if(EXISTS "${srcdir}/LICENSE")
+    file(COPY "${srcdir}/LICENSE" ${dstfile})
```

Comment:
Should it be `file(COPY "${srcdir/LICENSE" DESTINATION "${dstfile}")`? See docs: https://cmake.org/cmake/help/v3.3/command/file.html